app/services/burn_up_calculator.rb

```diff
@@ -6,24 +6,28 @@ def call(project)
 
     calculate_points
 
-    @group_by_day
+    @report_data
   end
 
   private
 
   attr_reader :service_full
 
   def initialize_group_by_day
-    @group_by_day = [
-      { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
-      { name: 'real',  data: service_full.group_by_day },
-      { name: 'ideal', data: service_full.group_by_day.dup }
-    ]
+    @report_data = {
+      group_by_day: [
+        { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
+        { name: 'real',  data: service_full.group_by_day },
+        { name: 'ideal', data: service_full.group_by_day.dup }
+      ],
+      total_backlog_points: total_backlog_points
+    }
   end
 
   def total_backlog_points
-    stories = service_full.instance_variable_get('@stories')
-    stories.map(&:estimate).compact.sum
+    stories = @service_full.instance_variable_get('@stories')
+    backlog_stories = stories.select { |story| story.state != 'accepted' }
```

Comment:
I think here you can replace `story.state != 'accepted' by '!sotry.accepted?'.

Reply:
Maybe this is not possible, because I checked the model and the schema, the state is `string`, this is not a `enum`.

Reply:
Actually it is possible, I'll make the change since it becomes easier to understand. Thanks.

Reply:
yeah. @patybastos we use a `state_machine` on the model.
https://github.com/Codeminer42/cm42-central-support/blob/master/lib/central/support/concerns/story_concern/transitions.rb#L9

So I think it implement those helpers.